src/analysis/retail/evoker/preservation/modules/core/MasteryEffectiveness.tsx

```diff
@@ -266,6 +235,73 @@ class MasteryEffectiveness extends Analyzer {
     this.totalMasteryHealingDone += absorbFromMasteryBonusUsed;
   }
 
+  __masteryBoostCheck(event: ApplyBuffEvent | RefreshBuffEvent, existingAbsorb: number) {
```

Comment:
rather than the __ prefix, these should just be private functions. editor should be able to mass-rename and then adding `private` to them will be easy
```suggestion
  private masteryBoostCheck(event: ApplyBuffEvent | RefreshBuffEvent, existingAbsorb: number) {
```